### DISCUSSION TITLE
Removing alias shadows

### DISCUSSION THREAD

**Discussion by shiningflint** [FelixKratz/SketchyBar#365] [Q&A] [answered]
![Screenshot 2023-04-24 at 12 04 29](https://user-images.githubusercontent.com/5021884/233893098-ab9d43b4-5d79-4fbb-9b76-751e4f780ea4.png)
Hi everyone!
is it possible to remove the shadows in an alias item?

I've tried to set shadow.drawing=off to all possible settings I thought I have found. (icon & label) still no luck.
I'm on ventura 13.3.1
```
sketchybar --default icon.font="Hack Nerd Font:Bold:17.0"  \
                     icon.color=${ACCENT_COLOR}                 \
                     icon.shadow.drawing=off                 \
                     icon.background.shadow.drawing=off                 \
                     label.shadow.drawing=off                 \
                     label.background.shadow.drawing=off                 \
                     label.font="Hack Nerd Font:Bold:14.0" \
                     label.color=${ACCENT_COLOR}                \
                     padding_left=5                        \
                     padding_right=5                       \
                     label.padding_left=4                  \
                     label.padding_right=4                 \
                     icon.padding_left=4                   \
                     icon.padding_right=4                  \
                     alias.color=${ACCENT_COLOR}
```

**Comment by FelixKratz** [comment, marked as answer]
I dont think the shadows are coming from sketchybar. I think the menu items have a shadow backed in, so there is nothing we can do, except of course recreating the item by hand in sketchybar.

**Reply by shiningflint** [reply]
Thank you for the answer!
Ideally I want to create like a notification center with custom counter for messaging apps I use.
But I have very limited experience, I don't know where to start. It also looks like I will need to dig into implementations of each application and how to get unread messages from them, seems impossible.

**Reply by FelixKratz** [reply]
If you just want to show the "badges" (as seen in the Dock), there is a simple way to get the badge count for an application as discussed here: https://github.com/FelixKratz/SketchyBar/discussions/317

Additionally it is possible to read the full notification content from the notification database, which is a bit more complicated but not impossible (example: https://github.com/crissNb/Dynamic-Island-Sketchybar)

**Reply by shiningflint** [reply]
Wow, thank you for the amazing tip!

I was able to prototype up a crude one
<details>
<summary>wechat.sh</summary>

```
WECHATINFO=$(lsappinfo info -only StatusLabel `lsappinfo find LSDisplayName=WeChat`)
COUNT=${WECHATINFO:25:1}

if [ $COUNT = "\"" ]; then
  DRAWING=off
else
  DRAWING=on
fi

sketchybar --set $NAME drawing=$DRAWING label="${COUNT}"
```
</details>
